ui/Makefile

```diff
@@ -40,12 +40,12 @@ start: deps
 .PHONY: test
 test: deps $(SOURCES)
 	@echo "+ $@"
-	cd apps/platform; yarn test
+	yarn test
```

Comment:
The top level `yarn test` will delegate to the `yarn test` in `apps/platform`, and calling from the top level lets us use the `CI=true` variable without needing to modify the lower level package.json.

In a future cleanup, we will need to do this when the top level package.json is removed.